editor/src/java/com/defold/editor/Start.java

```diff
@@ -117,20 +120,13 @@ private void prunePackages() {
     @Override
     public void start(Stage primaryStage) throws Exception {
         try {
-            /*
-              Note
-              Don't remove
-
-              Background
-              Before the mysterious line below Command-H on OSX would open a generic Java about dialog instead of hiding the application.
-              The hypothosis is that awt must be initialized before JavaFX and in particular on the main thread as we're pooling stuff using
-              a threadpool.
-              Something even more mysterious is that if the construction of the buffered image is moved to "static void main(.." we get a null pointer in
-              clojure.java.io/resource..
-            */
-
-            new BufferedImage(1, 1, BufferedImage.TYPE_3BYTE_BGR);
```

Comment:
This old hack doesn't work anymore, but using of `Desktop.getDesktop().setAboutHandler(null);` helps and CMD+H works fine